src/Sticky.jsx

```diff
@@ -276,6 +276,12 @@ class Sticky extends Component {
         this.update();
     }
 
+    componentDidUpdate(prevProps, prevState) {
+        if (prevState.status !== this.state.status) {
+            this.props.onStickyStateChange(this.state.status === 2);
```

Comment:
Maybe we can use `STATUS_ORIGINAL`, `STATUS_RELEASED`, and `STATUS_FIXED` as returned status, and export them as static constants.